assets/helpers/externalLinks.js

```diff
@@ -146,10 +150,20 @@ function getSubsLinks(
 
 }
 
+// Builds a link to the digital pack checkout.
+function getDigitalCheckout(referrerAcquisitionData: ReferrerAcquisitionData): string {
```

Comment:
I think we should include a promo code to identify the source of this redirect in the subs site

Reply:
Ok I've updated this, I *think* that query param is how subs frontend picks this up on the checkouts.